pkg/kubectl/cmd/create_podsecuritypolicy.go

```diff
@@ -0,0 +1,171 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	pspv1beta1 "k8s.io/api/extensions/v1beta1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgo "k8s.io/client-go/kubernetes/typed/extensions/v1beta1"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/util/i18n"
+)
+
+var (
+	PSPLong = templates.LongDesc(i18n.T(`
+		Create a PodSecurityPolicy.`))
+
+	pspExample = templates.Examples(i18n.T(`
+
+		# Create a PodSecurityPolicy named "example" that simply uses default values.
+		kubectl create podsecuritypolicy example
+
+		# Create a PodSecurityPolicy named "my-psp"
+		kubectl create podsecuritypolicy privileged --selinux=MustRunAs
```

Comment:
privileged -> my-psp

Reply:
This example could lead to frustration -- `MustRunAs` typically also requires some additional fields (ranges/levels/etc). While it's possible to create a PSP with this strategy but without required fields, it fails later and fails badly: https://github.com/kubernetes/kubernetes/issues/61179

I'd suggest to modify example to `--runasuser=RunAsAny`. Here I also use `runasuser` as it's more frequently used and demanded by users.

Reply:
Now I see that `RunAsAny` is a default value, so there is no value to explicitly provide it. Maybe just remove the 2nd example?